assets/js/src/core/modules/data-object/editor/shared-tab-manager/tabs/preview/preview-view.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React from 'react'
+import { useElementContext } from '@Pimcore/modules/element/hooks/use-element-context'
+import { getPrefix } from '@Pimcore/app/api/pimcore/route'
+import { useTranslation } from 'react-i18next'
+
+export const PreviewView = (): React.JSX.Element => {
+  const { t } = useTranslation()
+  const { id } = useElementContext()
+
+  return (
+    <iframe
+      height="100%"
```

Comment:
Can we please replace the hardcoded width="100%" and height="100% with our css classes (`w-full h-full`)?